test/e2e-tests.sh

```diff
@@ -86,6 +86,38 @@ fi
 kubectl -n ${SYSTEM_NAMESPACE} patch configmap/config-autoscaler --type=merge --patch='{"data":{"allow-zero-initial-scale":"true"}}'
 add_trap "kubectl -n ${SYSTEM_NAMESPACE} patch configmap/config-autoscaler --type=merge --patch='{\"data\":{\"allow-zero-initial-scale\":\"false\"}}'" SIGKILL SIGTERM SIGQUIT
 
+kubectl -n "${SYSTEM_NAMESPACE}" patch configmap/config-leader-election --type=merge \
+  --patch='{"data":{"enabledComponents":"controller,hpaautoscaler,webhook", "buckets": 10}}'
```

Comment:
I guess controller & hpaautoscaler currently don't support buckets, since it hasn't been merged, but do we want to enable that here before they do?

Reply:
I want to exercise the bucketing in the webhook, and it is harmless for the others until they switch to the new style of enabling LE.